milton-processor/package.json

```diff
@@ -15,20 +15,21 @@
     "start": "node ./index.js",
     "gcp-build": "tsc -p .",
     "deploy-appengine": "gcloud app deploy",
-    "deploy-function": "gcloud functions deploy scribe --runtime nodejs10 --allow-unauthenticated --trigger-http --memory 512MB --service-account scribe@dsouza-proving-ground.iam.gserviceaccount.com --env-vars-file env.yaml"
+    "deploy-function": "gcloud functions deploy scribe --runtime nodejs10 --allow-unauthenticated --trigger-http --memory 8192MB --timeout 5m --service-account scribe@dsouza-proving-ground.iam.gserviceaccount.com --env-vars-file env.yaml"
   },
   "dependencies": {
     "@google-cloud/storage": "^5.7.0",
-    "@mozilla/readability": "github:mozilla/readability",
+    "@mozilla/readability": "^0.4.2",
     "base64url": "^3.0.1",
+    "chrome-aws-lambda": "^10.1.0",
```

Comment:
This includes a full Chrome binary, right? Did you check whether it changes the cold start time at all?

Reply:
Yea, it was relatively slow till I [16x'd the cloud function size](https://github.com/MattX/Milton/pull/43/files#diff-e22e7391650316577dcf30cc1d8baf280d3ee6d9e25de0f4819aedf15b3810d1R18), but now seems roughly analogous to what it was prior to this change. We'll see if this takes me out of the free tier (from what I can tell it's still fine because of how low traffic to the function is), but I've been wanting this snapshotting functionality anyways, so don't mind as long as it isn't super expensive.